src/XMLimport.h

```diff
@@ -58,6 +58,8 @@ class XMLimport : public QXmlStreamReader
     std::pair<dlgTriggerEditor::EditorViewType, int> importFromClipboard();
 
 private:
+    const QString YES = QStringLiteral("yes");
```

Comment:
:+1: Yeah - or rather `YES` - must confess I am doing this in another PR I have in progress in a different class - and the only thing I would suggest is to move this to the top of the `.cpp` file so that it is visible in the same file as it is used. 😆 

TBH *Any* `QStringLiteral(...)` that is used more than once will benefit from this...

![image](https://user-images.githubusercontent.com/6163092/103178333-cfd4bb00-4879-11eb-876a-c5dc3215e68a.png)

Reply:
I don't see the benefits it brings and I'm not convinced by the proposed performance improvements. I think it just makes the code harder to read because you have to go look up what these abbreviations mean.

Reply:
I am Java dev - Java String Pool ❤️  
Sure, will move that to top (although initially had some compilation error, will check that)

Reply:
> I don't see the benefits it brings and I'm not convinced by the proposed performance improvements.

I refer you to https://www.qt.io/blog/2014/06/13/qt-weekly-13-qstringliteral - particularly "Avoid duplicate QStringLiterals"

Reply:
1727954, before constant, 69.78M
8d72689, after constant, 69.79M

There are no significant space savings to speak of, but the code is more complicated because it involves macros. I think it's a lose-lose. We can keep this usage here because the pattern is often repeated, but let's not add it to other classes.